.github/workflows/create-issue.yml

```diff
@@ -0,0 +1,34 @@
+on: push
+  issues:
+    types: [opened]
+name: Create new tickets in Jira
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Create new tickets in Jira
+    steps:
+      - name: Login to Jira
+        uses: acquia/gajira-login-2@server
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+      - name: Create Jira ticket
+        id: create
+        uses: acquia/gajira-create@server
+        with:
+          project: DX
+          issuetype: Task
+          summary: ${{ github.events.issue.title }}
+          description: ${{ github.events.issue.url }}
+          fields: '{"components": [{"id": "21989"}]}'
+      - name: Log created issue
+        run: echo "Issue ${{ steps.create.outputs.issue }} was created"
+      - name: Update Github issue with Jira ticket prefix
+        uses: actions-cool/issues-helper@v3
```

Comment:
So if I create a new issue "Add cool feature foo", this will create a Jira ticket, then automatically change the title to "CLI-123: Add cool feature foo"? Pretty slick! But I feel like this shouldn't require a third-party action. Have you looked into whether Actions can manage issues natively?

I just worry about getting approval for that particular action.

Reply:
Yes, that's what it does. 

I couldn't find a native action to do this.

Reply:
I suggested a change to avoid using a third-party action. See also https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows

You'll probably need to set issue write permissions on the token, same as perform.